subprojects/analysis-kotlin-descriptors/compiler/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/descriptors/compiler/configuration/resolve/DokkaJsResolverForModuleFactory.kt

```diff
@@ -42,6 +42,10 @@ internal class DokkaJsResolverForModuleFactory(
         )
     }
 
+    @Deprecated(
```

Comment:
These modules are 100% internal, I think there's no real benefit in deprecating things, and they can just be removed. If you delete it, make sure to also delete the other deprecated `createResolverForModule`, for some reason there are two..